packages/crayons-icon/readme.md

```diff
@@ -0,0 +1,69 @@
+[![](https://data.jsdelivr.com/v1/package/npm/@freshworks/crayons-icon/badge)](https://www.jsdelivr.com/package/npm/@freshworks/crayons-icon)
+# Icon Library (@freshworks/crayons-icon)
+
+**Freshworks Crayons** publishes `@freshworks/crayons-icon` as an `Icon Library`. Following features are made available to users via the Lib utils.
+
+1. Optimized set of Crayons Icons in `dist/icons` path. You are free to use them for project related purposes.
+2. `Crayons` SVG Icons are also available as an esm module. Supports Tree-Shaking. This is a useful feature especially for customization purposes.
+3. Enables `@freshworks/crayons-icon` CLI Interface via which you can leverage the SVGO Tooling to optimize your project svg-icons.
+4. Crayons Iconlib default `svgo config` (svg compression util) is available as part of `dist` folder. Refer usage docs on how to use it in your project.
+
+<details> 
+  <summary>Click to see the List of Icon Assets available as part of Repo.</summary>
+   <img alt="Crayons Icon Assets" width="100%" src="docs/crayons-icon-cli/crayons-icon-assets.jpg" />
+</details>
+
+## Importing Icons from '@freshworks/crayons-icon'. [ Supports Tree-Shaking ].
+
+You can also import Crayons Icons as an esm Module.
+
+1. HTML USAGE
+
```

Comment:
Can you pls follow the Tab approach for displaying code . Using `code-group`

Reply:
Fixed

Reply:
@arvindanta I am using `code-group` in icon README.md. But for Github Readme , I cannot use the same as its disturbing the formatting and we need this ReadMe for npm registry package listing.
Here is what happens when I use code-group in root github file.
<img width="1175" alt="Screenshot 2021-12-20 at 4 10 19 PM" src="https://user-images.githubusercontent.com/76547218/146755116-9bd0e27a-5dd6-427e-91fa-835a8199c554.png">

And here is what we need as Lib Readme.I have other docs files to show with arrow section titled 'Click to see....'
https://github.com/freshworks/crayons/tree/crayons-icon-cli/packages/crayons-icon

Marking it as closed.